fe-nc-news/src/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import getComments from './utils/GetComments'
+import {React, useState, useEffect} from 'react'
+
+const Comments = ({article_id}) => {
+ const [comments, setComments] = useState([])
+ useEffect(()=>{
+    getComments(article_id).then((commentData)=>{
+        setComments(commentData)
+    })
+    }, [article_id])
+return (
+<>
+<h3>Comments</h3>
+    <div className="commentGrid">
+    {comments.length > 0 ?(
+    
+    comments.map((comment)=>{
```

Comment:
You could look at destructuring inline in your map function
eg ` comments.map(({body, author etc}) => {...`
Just something for you to think about 😄